parcellab/common/templates/_pod.tpl

```diff
@@ -45,11 +50,19 @@ spec:
       {{- toYaml .volumeTemplate | nindent 6 }}
     {{- end }}
     {{- end }}
-    {{- if and .Values.datadog .Values.datadog.enabled }}
+    {{- if and $datadog $datadog.enabled }}
     - name: apmsocketpath
       hostPath:
         path: /var/run/datadog/
     {{- end }}
+  {{- if $initContainers }}
+  {{- range $initContainers }}
+  initContainers:
+    {{- include "common.container"
```

Comment:
Aligned to the previous comment in the container file. I would put the include in every list element
```suggestion
    - {{- include "common.container"
```